topo/flat/one/corp_conf.yaml

```diff
@@ -0,0 +1,19 @@
+dps:
```

Comment:
This should be parameterized, not different explicit topos for each case, there's simply too much redundancy -- that's just too unwieldy.  See testing/python_lib/build_config.py for the basics of that, but it should be extended to support these flat topologies as well.

Reply:
Using build_config.py to generate Faucet configs for corp and faucet network now. PTAL..